roles/cifmw_cephadm/tasks/configure_object.yml

```diff
@@ -44,15 +44,15 @@
       KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     output_dir: "/home/zuul/ci-framework-data/artifacts"
     script: |-
-      oc rsh openstackclient openstack service create --name swift --description 'OpenStack Object Storage' object-store
-      oc rsh openstackclient openstack user create --project service --password {{ cifmw_ceph_rgw_keystone_psw }}  swift
-      oc rsh openstackclient openstack role create swiftoperator
-      oc rsh openstackclient openstack role create ResellerAdmin
-      oc rsh openstackclient openstack role add --user swift --project service member
-      oc rsh openstackclient openstack role add --user swift --project service admin
-      oc rsh openstackclient openstack endpoint create --region regionOne object-store public http://{{ cifmw_cephadm_vip }}:8080/swift/v1/AUTH_%\(tenant_id\)s
-      oc rsh openstackclient openstack endpoint create --region regionOne object-store internal http://{{ cifmw_cephadm_vip }}:8080/swift/v1/AUTH_%\(tenant_id\)s
-      oc rsh openstackclient openstack role add --project admin --user admin swiftoperator
+      oc -n openstack rsh openstackclient openstack service create --name swift --description 'OpenStack Object Storage' object-store
```

Comment:
is the `namespace` something we want to pass as variable? This would cover (in the future) the deployment on a project != `openstack` || multiple ctlplane in different namespaces.

Reply:
Good idea , Updated in the new patchset.